build/dependencies.props

```diff
@@ -8,7 +8,7 @@
     <InternalAspNetCoreSdkPackageVersion>2.2.0-preview1-17090</InternalAspNetCoreSdkPackageVersion>
     <MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>2.8.0</MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>
     <MicrosoftExtensionsDependencyModelPackageVersion>2.2.0-preview1-26618-02</MicrosoftExtensionsDependencyModelPackageVersion>
-    <MicrosoftNETCoreApp22PackageVersion>2.2.0-preview1-26618-02</MicrosoftNETCoreApp22PackageVersion>
+    <MicrosoftNETCoreApp30PackageVersion>2.2.0-preview1-26618-02</MicrosoftNETCoreApp30PackageVersion>
```

Comment:
This package doesn't actually have netcoreapp30 does it?

Reply:
Correct, it does not. But netcoreapp2.2 should be compatible with netcoreapp3.0. Once we get a full round of 3.0 stuff, UpdateRepos will adjust this correctly.